api/src/routing.js

```diff
@@ -151,6 +153,16 @@ app.get('/', function(req, res) {
   }
 });
 
+app.get('/favicon.ico', (req, res) => {
+  db.medic.get('branding').then(doc => {
+    db.medic.getAttachment(doc._id, doc.resources.favicon).then(blob => {
```

Comment:
The name is hardcoded here, right? So we shouldn't need to get the `branding` doc at all. eg:

```js
db.medic.getAttachment('branding', 'favicon').then(blob => {
```

Reply:
Hmm attachment are named based on file name so `favicon` above should be `favicon.ico` so if a partner decide to  upload a favicon with the name `lg.ico` we will have to get that name from the `doc.resources` which keeps track of the attachments. That is why I am fetching the doc and getting the name of the  attachment `doc.resources.favicon` i.e. this is key value based

Reply:
Ahh, that makes sense. Good call.